packages/flutter_tools/test/general.shard/android/android_gradle_builder_test.dart

```diff
@@ -31,48 +28,15 @@ void main() {
   group('gradle build', () {
     BufferLogger logger;
     TestUsage testUsage;
-    MockAndroidSdk mockAndroidSdk;
-    MockAndroidStudio mockAndroidStudio;
-    FakePlatform android;
     FileSystem fileSystem;
-    Cache cache;
     FakeProcessManager processManager;
 
     setUp(() {
       processManager = FakeProcessManager.list(<FakeCommand>[]);
       logger = BufferLogger.test();
       testUsage = TestUsage();
       fileSystem = MemoryFileSystem.test();
-      mockAndroidSdk = MockAndroidSdk();
```

Comment:
This mocking is not necessary since the logic for locating the gradle wrapper is provided by GradleUtils, which is injected and is faked for the test